src/Inciter/Performer.C

```diff
@@ -13,6 +13,7 @@
 // *****************************************************************************
 
 #include <string>
+#include <math.h>
```

Comment:
`#include <cmath>` is a better choice for C++ code, since that defines all functions in a namespace to avoid name collisions. But is this `include` really needed here.